src/main/kotlin/com/coder/gateway/CoderSettingsConfigurable.kt

```diff
@@ -149,6 +149,12 @@ class CoderSettingsConfigurable : BoundConfigurable("Coder") {
                     .bindText(state::workspaceFilter)
                     .comment(CoderGatewayBundle.message("gateway.connector.settings.workspace-filter.comment"))
             }.layout(RowLayout.PARENT_GRID)
+            row(CoderGatewayBundle.message("gateway.connector.settings.default-ide")) {
+                textField().resizableColumn().align(AlignX.FILL)
+                .bindText(state::defaultIde)
+                    .comment("The default IDE version to display in the IDE selection dropdown. " +
```

Comment:
I wonder if we need to make mention that JetBrains by default only shows the latest version (latest stable and latest EAP I think, right?), so as soon as JetBrains updates their list this setting may become obsolete unless they are using just the IDE name or year.

But, maybe this option will only be used by folks that have provided their own list instead of fetching from JetBrains, so that might not be an issue.